src/cache/precomputed_quantities.jl

```diff
@@ -137,9 +136,9 @@ function precomputed_quantities(Y, atmos)
             ᶜdetrʲs = similar(Y.c, NTuple{n, FT}),
             ᶜturb_entrʲs = similar(Y.c, NTuple{n, FT}),
             ᶠnh_pressure₃ʲs = similar(Y.f, NTuple{n, C3{FT}}),
-            ᶜgradᵥ_θ_virt⁰ = Fields.Field(C3{FT}, cspace),
-            ᶜgradᵥ_q_tot⁰ = Fields.Field(C3{FT}, cspace),
-            ᶜgradᵥ_θ_liq_ice⁰ = Fields.Field(C3{FT}, cspace),
+            ᶜgradᵥ_θ_virt⁰ = similar(Y.c, C3{FT}),
+            ᶜgradᵥ_q_tot⁰ = similar(Y.c, C3{FT}),
+            ᶜgradᵥ_θ_liq_ice⁰ = similar(Y.c, C3{FT}),
```

Comment:
Any particular reason that we're using `similar` here over `Field`? I kind of prefer `Field` over `similar`, since those constructors don't actually need the field values in `Y.c`. (I was thinking that we could refactor in the other direction).

Reply:
I'm ambivalent about using `Field` or `similar`. I was just enforcing consistency with this change, but we can hold off on that until a future PR.